po/pl.po

```diff
@@ -1882,17 +1883,17 @@ msgstr ""
 "nieudane: %s"
 
 #: rust/libnewsboat/src/filterparser.rs:63
-#, fuzzy
 msgid "attribute name"
-msgstr "atrybut nie znaleziony"
+msgstr "nazwa atrybutu"
 
 #: rust/libnewsboat/src/filterparser.rs:64
 msgid "one of: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
-msgstr ""
+msgstr "jeden z: =~, ==, =, !~, !=, <=, >=, <, >, between, #, !#"
 
 #: rust/libnewsboat/src/filterparser.rs:65
+#, fuzzy
```

Comment:
Why was this marked fuzzy? The translation looks complete and correct to me (but I don't speak Polish, so might be totally wrong).

Reply:
I didn't have time to look into sources and check if options on the list (quoted string, range, number) should be translated or not

Reply:
Ah. Yeah, they should be translated (unlike `between` in a previous string, which is an operator).